walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,61 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+  
+  /** Creates a query that allows for different possible meeting time ranges to be found 
+    * given any request for a meeting throughout a day
+    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
This algorithm will not work in all cases. Add a test case to test a list of events which has an afternoon meeting as the first event in the collection, then a morning meeting next